config/config.yaml

```diff
@@ -5,3 +5,8 @@ admincentral_cf_bucket: "bootstrap-awss3cloudformationbucket-19qromfd235z9"
 template_key_prefix: {{ environment_variable.TRAVIS_BRANCH | default("testing") }}
 toil_key_pair: "toil"
 toil_ami: "ami-024a64a6685d05041" #ubuntu 18.04
+stack_tags:
+  Department: "SysBio"
+  Project: "amp-ad"
+  OwnerEmail: "tess.thyer@sagebase.org"
```

Comment:
Can OwnerEmail be a group rather than an individual?

Reply:
Or perhaps this should be @wpoehlm? IBC/DPE is going to maintain this for him, but it may not be me, but rather any one of our engineers. But Will's the one using this infra and so may have a better idea what cost center to use.

Reply:
@zaro0508 @brucehoff We'd like to use a group email for OwnerEmail, if that is acceptable. @thomasyu888 will be in touch about that later.

Reply:
The google group is dpe@sagebase.org

Reply:
done